Docs/reference/apidocs/models.md

```diff
@@ -1194,6 +1195,14 @@ Models:
 | shellRequestId | [SingularityTaskShellCommandRequestId](models.md#model-SingularityTaskShellCommandRequestId) | optional |  |
 
 
+## <a name="model-SingularityTaskWebhook"></a> SingularityTaskWebhook
```

Comment:
A quick note that everything in the apidocs folder here is auto generated by swagger for us. What you'll want to do is add swagger annotations to the object, similar to what you see [here](https://github.com/HubSpot/Singularity/blob/master/SingularityBase/src/main/java/com/hubspot/singularity/SingularityRequest.java#L193). That should get swagger to include it in the models docs for us